source/material_model/rheology/visco_plastic.cc

```diff
@@ -479,6 +479,20 @@ namespace aspect
               composition_mask.set(i,false);
           }
 
+        // These are the compositional fields used to track deposition depth and
+        // sediment age of the sediments deposited by the FastScape plugin.
+        // ToDo: insert check that these fields are only set as a mask when fastscape is used?
```

Comment:
Just convention:
```suggestion
        // TODO: insert check that these fields are only set as a mask when fastscape is used?
```
And I would remove this TODO and check. Someone could create a field named "sediment_age" without intending to use Fastscape, and it would come as a surprise if the model fails then.